Fix profiler_kineto Clang errors

sraikund16 opened the pull request:
Summary: There are clang errors in profiler_kineto. It would probably be a good idea to fix them as the file is already quite dense.

Test Plan: Make sure all on Phabricator all tests under static_tests/lint_root pass

Differential Revision: D58431005
@diff-train-skip-merge

aaronenyeshi commented on torch/csrc/autograd/profiler_kineto.cpp:
```diff
@@ -47,8 +47,7 @@ __attribute__((weak)) int acc_get_device_type() {
 namespace torch {
-namespace autograd {
-namespace profiler {
+namespace autograd::profiler {
```
do we need to combine these? I feel the normal convention in pytorch separates each namespace into their own lines

sraikund16 replied:
yes it fails lint_root with modernize-concat-nested-namespaces

sraikund16 replied:
We can do whichever you think is more important but I'd rather get rid of all the static errors whenever a diff touches this file

aaronenyeshi reviewed (changes requested):
Please check lintrunner error

sraikund16 committed:
Fix profiler_kineto Clang errors (#128464)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/128464

There are clang errors in profiler_kineto. It would probably be a good idea to fix them as the file is already quite dense.

Test Plan: Make sure all on Phabricator all tests under static_tests/lint_root pass

Reviewed By: aaronenyeshi

Differential Revision: D58431005

aaronenyeshi reviewed (approved):
LGTM!
